frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -184,15 +184,11 @@ const getApiEvaluations = (property: IProperty): IEvaluation[] => {
   evaluations.push({
     parcelId: isParcelOrSubdivision(property) ? property.id : undefined,
     buildingId: property.propertyTypeId === PropertyTypes.BUILDING ? property.id : undefined,
-    value: isParcelOrSubdivision(property)
-      ? property.assessedLand || 0
-      : property.assessedBuilding || 0,
-    date: isParcelOrSubdivision(property)
-      ? property.assessedLandDate ?? formatDate(new Date())
-      : property.assessedBuildingDate ?? formatDate(new Date()),
-    rowVersion: property.assessedLandRowVersion,
+    value: property.assessedBuilding || 0,
+    date: property.assessedBuildingDate ?? formatDate(new Date()),
+    rowVersion: property.assessedBuildingRowVersion,
     key: EvaluationKeys.Improvements,
-    firm: property.assessedLandFirm ?? '',
+    firm: property.assessedBuildingFirm ?? '',
```

Comment:
An `Improvement` should always be `assessedBuilding*`.  @devinleighsmith I'm not certain this is implemented correctly through the application.

Reply:
In my PR I made the assumption that improvements were specific to land, and that a regular assessment evaluation on a building would be the assessedbuilding.

Reply:
sounds like the opposite is true, that an improvement is always an assessedbuilding and that buildings aren't allowed to use EvaluationKeys.assessed

Reply:
My assumption was based around how assessments for buildings previously worked. Did we create a migration to move all of the old building assess values over to improvements?